Atos20/indecision-maker-server
# Card games setup ng

#### What's this PR do?  

+ Deletes old seed file and migration file for students
+ Adds a migration file to set up the schema for the `card_games` table
+ Adds a `data` directory that contains a file for `cardGamesData`, which exports an array of card game objects
+ Adds a seed file for all card games, using the exported array from `cardGamesData` to insert a new card game object into the `card_games` table

#### Where should the reviewer start?  

It would be best to start in `data/cardGameData.js` to analyze the structure of the objects.  Then, review `cardgames.js` (seed file) to see how the data from the data directory is used to seed data in the table.  

#### How should this be manually tested?  

This can be tested by pulling down these changes and running `knex migrate:latest` and `knex seed:run` from the terminal.  As long as the database is hosted locally, you should be able to use Postico to view the data in the `card_games` table.

#### Any background context you want to provide?

 I am realizing the naming conventions for the files are not consistent, so I apologize for that.  For example, `cardGamesData.js` and `cardgames.js` don't show consistent capitalization of the letter `G` in `games`.  I have changed this to `cardGames.js` locally but can't seem to be able to commit this change.  Will look into this further tomorrow. 

#### What are the relevant tickets?  

Closes #12 

#### Screenshots (if appropriate) 

![Screen Shot 2020-10-28 at 9 20 26 PM](https://user-images.githubusercontent.com/62262404/97521559-6b71aa00-1963-11eb-9140-e95a6f71af47.png)
